doc/documentation/prefetching.md

```diff
@@ -0,0 +1,72 @@
+---
+title: Prefetching
+description: Prefetch responses before they're requested.
+---
+
+SPF is designed to enable fast navigation, and the best way to
+speed up a navigation request is to not make it all.
+Prefetching allows you to fetch responses before they are
+requested and store them in the [local cache][] until they are
+needed.  You can also prefetch [scripts and styles][resources]
+to prime the browser cache.
+
+
+## Prefetching requests
+
+Prefetching a response is done by calling the [spf.prefetch][]
+function, which behaves nearly identically to [spf.navigate][]
+and accepts the same [callbacks and cancellations][] when passed
+an object that conforms to the [spf.RequestOptions][] interface.
+A list of callbacks follows:
+
+| Callback    | State                         | Cancel |
+|:------------|:------------------------------|:-------|
+| `onRequest` | Started; Sending Prefetch     | Abort  |
+| `onProcess` | Processing; Response Received | Abort  |
+| `onDone`    | Done                          |        |
+
+When SPF sends the prefetch request to the server, it will
+append the configurable identifier to the URL in the same manner
+as navigation; by default, this value will be `?spf=prefetch`.
+
+When the prefetched response has been
```

Comment:
super nit: can you re-fill these lines so they are more even? If that isn't something your editor can do very easily don't worry about it.